src/containters/Home/components/HeroView/HeroView.js

```diff
@@ -4,15 +4,14 @@ import { IoIosArrowForward } from 'react-icons/io';
 import { useEffect, useState } from 'react';
 import { heroView } from '../../services/heroView';
 import { BrowserRouter as Router, Link } from 'react-router-dom';
-const HeroView = () => {
-    const [heroViewData, setHeroViewData] = useState({});
-
+const HeroView = (props) => {
```

Comment:
- Please pass the object directly as a prop no need to pass an object containing only single object.
- Also use prop types library to define the prop types of the props for the  **HeroView**  component and please add them at the end of the script of the **HeroView** component as this is the common convention for this (Hint: install the prop-types package as it is a stand alone package).